test/sanity/issue6113-mac-create-crash/test.py

```diff
@@ -0,0 +1,30 @@
+import time
+import os
+import platform
+import sys
+
+if platform.system() != 'Darwin':
+    print 'Skipped for non Mac platform'
+    sys.exit(0)
+
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
+
+chrome_options = Options()
+chrome_options.add_argument("nwapp=" + os.path.dirname(os.path.abspath(__file__)))
+
+driver = webdriver.Chrome(executable_path=os.environ['CHROMEDRIVER'], chrome_options=chrome_options)
+driver.implicitly_wait(2)
+try:
+    handles = driver.window_handles
+    print handles
+    driver.switch_to.window(handles[1])
```

Comment:
There is a race condition here: the number of windows/handles may not be 2 when this step is reached. consider using wait_for_window_handles in nw_utils.

Reply:
Added wait_window_handles in test.py

Reply:
> Added wait_window_handles in test.py

I don't see any update in this PR.